{"name": "test_login_with_invalid_email_and_valid_password", "status": "passed", "start": 1722274056911, "stop": 1722274058204, "uuid": "3ab0ce35-3b2b-4ed3-9686-30eaabd39702", "historyId": "759fa7c0406bbfbdfa09753295718a23", "testCaseId": "759fa7c0406bbfbdfa09753295718a23", "fullName": "tests.test_loging.TestLoging#test_login_with_invalid_email_and_valid_password", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_loging"}, {"name": "subSuite", "value": "TestLoging"}, {"name": "host", "value": "kengni"}, {"name": "thread", "value": "3344098-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_loging"}]}